libs/application/templates/marriage-conditions/src/lib/MarriageConditionsTemplate.ts

```diff
@@ -16,7 +16,6 @@ import {
   DistrictsApi,
 } from '@island.is/application/types'
 import { assign } from 'xstate'
-import { Features } from '@island.is/feature-flags'
 import { getSpouseNationalId } from './utils'
 import { FeatureFlagClient } from '@island.is/feature-flags'
```

Comment:
What about this and L#60-70 ?

Reply:
keeping this for now, 
for testing on dev with fake data

Reply:
Okay, I see. Fake Data sounds like something that should always be an option in the dev environment and I feel like the feature flags is not the correct tool to use for that. You should look into the parental leave application, that renders on dev a prerequisite subsection that asks the user if he wants to use fake data or not.

Reply:
@saevarma 

Okay, parental leave application, however, is using the deprecated `isRunningOnEnvironment` check. The comment above it says "Any decisions based on the environment name should be rather based on feature flags and/or configuration." 

I would also want to be able to clean up feature flags thats are used for fake data, but im rather unsure at this point what the best way is. Perhaps good food for thought, to establish what we want to continue with.

Reply:
I missed that comment. Thanks for clarifying. I will take this up with Norda about general strategy on fake data in dev environment.